### ISSUE TITLE
Migration functions should mention `using Primes`

### ISSUE THREAD

**Issue by TotalVerb** [JuliaLang/julia#18831]
A user was confused today because he tried `Pkg.add("Primes")`, as instructed, but that does not fix the issue. The migration message should mention `using Primes`.

``` jlcon
julia> factor(100)
ERROR: factor(100,) has been moved to the package Primes.jl.
Run Pkg.add("Primes") to install Primes on Julia v0.5-
 in factor(::Int64, ::Vararg{Int64,N}) at ./deprecated.jl:210
```

**Comment by fredrikekre** [comment]
Related:

```jl
julia> using Primes

julia> primes(1,2,3)
ERROR: primes(1,2,3) has been moved to the package Primes.jl.
Run Pkg.add("Primes") to install Primes on Julia v0.5-
 in primes(::Int64, ::Int64, ::Int64, ::Vararg{Int64,N}) at ./deprecated.jl:210
```

since `primes(args...)` (all possible input arguments) is deprecated it doesn't show close candidates etc.

**Comment by lbollar** [comment]
In general, should the using statement be included in these kind of deprecation warnings? I just submitted a pull request for a similar type of thing, and I am wondering if this should be added.

**Comment by tkelman** [comment]
closed for Primes by #19164. maybe the other similar messages for Combinatorics, PkgDev etc should also have the same clarification added though?
